frontend/src/pages/GettingStarted.tsx

```diff
@@ -86,7 +107,18 @@ cssRaw(`
 }
 `);
 
+const demoPipelines = [
+  '[Demo] Unified DSL - Taxi Tip Prediction Model Trainer',
```

Comment:
latest name is not "Unified DSL" but "TFX pipeline" or something, please check the latest codes.

Reply:
Thanks! Updated referring to https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/config/sample_config.json